controller/failover/cluster.go

```diff
@@ -12,11 +12,20 @@ import (
 	"github.com/RocksLabs/kvrocks_controller/metadata"
 	"github.com/RocksLabs/kvrocks_controller/storage"
 	"github.com/RocksLabs/kvrocks_controller/util"
+	"github.com/RocksLabs/kvrocks_controller/config"
 )
 
+type ClusterConfig struct {
```

Comment:
[note for reviewers] not sure if this is the best place for this struct, I am contemplating whether I should keep it here or withing the new config package which I have made.